tests/mock_network/secure_message_delivery.rs

```diff
@@ -109,6 +109,7 @@ fn message_with_invalid_security(fail_type: FailType) {
 }
 
 #[test]
+#[ignore] // We cannot create a message with invalid sig FIXME (possibly we want to do so via mock)
```

Comment:
Note that when https://github.com/maidsafe/routing/pull/2135 is merged, this whole module is gone and is instead replaced with unit tests, so faking an invalid message would be easier there.